android/src/main/java/com/swmansion/rnscreens/NoOpFragment.kt

```diff
@@ -0,0 +1,27 @@
+package com.swmansion.rnscreens.gamma.helpers
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+
+class NoOpFragment: Fragment() {
```

Comment:
```suggestion
class AutoRemovingFragment: Fragment() {
```

Technically, it's not an noop, it's a concrete behaviour. Let's reflect that in the name.

Reply:
Changed in [ad55264](https://github.com/software-mansion/react-native-screens/pull/3089/commits/ad55264b4fe84d841f67aec23d68614e7240bbd2)